web/router.go

```diff
@@ -29,12 +29,15 @@ func NewRouter(settings RouterSetting) (model.Router, error) {
 	var err error
 	authorizer := authorization.NewAuthorizer()
 
+	r.APIRouterPath = "/api"
+	r.WebRouterPath = "/web"
+
 	r.APIRouter, err = api.NewRouter(
 		settings.Server,
 		settings.Logger,
 		authorizer,
 		settings.LoggerSettings,
-		settings.APIRouterSettings...,
+		append(settings.APIRouterSettings, api.WebRouterPrefixOption(r.WebRouterPath))...,
```

Comment:
Add web router prefix for API

Reply:
I believe all modifications to identifo itself is better to extract to separate PR, otherwise, it is very hard to review a part of the whole 160 files changes.

Reply:
Ok, I will move changes to separate PR.